src/state/wallets/selectors.ts

```diff
@@ -93,10 +93,13 @@ export const transactions = createSelector(wallets, electrumXSelectors.blockHeig
         }
 
         const isFromMyWalletTx = wallet.weOwnAddress(transaction.inputs[0].addresses[0]);
+
         let toExternalAddress;
         let toInternalAddress;
 
         const toAddress = transaction.outputs[0].addresses[0];
+        const isToMyWalletTx = wallet.weOwnAddress(toAddress);
+
         const isToInternalAddress = walletsList.some(w => w.weOwnAddress(toAddress));
 
```

Comment:
could `isToMyWalletTx` replace `isToInternalAddress` in other places as well?

Reply:
it is only used here to duplicate transactions done to the wallet itself. In the app, the internal transaction is a transaction done to any wallet that is also in the app